OWASP-Suppression.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.2.xsd">
+    <suppress>
+        <notes><![CDATA[
+      file name: echo-filter.jar (project :repose-aggregator:components:filters:echo-filter)
```

Comment:
What?

Reply:
Why is our echo filter being flagged, you mean? It looks like it's a false-positive. The plugin sees some properties of this dependency contain "echo" and thinks the dependency is something that it isn't. So here we tell the plugin that our echo filter is not the echo filter it thinks it is.